runtime/jcl/common/sun_misc_Unsafe.cpp

```diff
@@ -788,6 +788,58 @@ Java_jdk_internal_misc_Unsafe_objectFieldOffset1(JNIEnv *env, jobject receiver,
 	return offset;
 }
 
+void JNICALL
+Java_jdk_internal_misc_Unsafe_writebackMemory(JNIEnv *env, jobject receiver, jlong addr, jlong len)
```

Comment:
Does the method need to do any validation to ensure that the addr+length is an integral number of cachelines?

Reply:
I don't believe so. The Intel instruction manual says `the cache line that contains the
linear address specified with the memory operand.`

Reply:
@DanHeidinga made a change here to ensure that any partial cache lines are also written back, similar to the example here: https://lore.kernel.org/patchwork/patch/557913/ 

I believe this is ready for a re-review now.